packages/components-js/tests/vrt/playwright/specs/states/radio-button-wrapper.vrt.ts

```diff
@@ -73,14 +71,14 @@ const scenario = async (page: Page, theme: Theme, scheme?: PrefersColorScheme):
     prefersColorScheme: scheme,
   });
 
-  await forceHoverState(page, '.hover p-radio-button-wrapper input[type="radio"]');
-  await forceHoverState(page, '.hover .force-label > p-radio-button-wrapper >>> span');
+  await forceHoverState(page, '.hover p-radio-button-wrapper:not(.force-label) input[type="radio"]');
+  await forceHoverState(page, '.hover p-radio-button-wrapper.force-label >>> span');
   await forceHoverState(page, '.hover p-radio-button-wrapper span a');
   await forceFocusState(page, '.focus p-radio-button-wrapper input[type="radio"]');
   await forceFocusState(page, '.focus p-radio-button-wrapper span a');
-  await forceFocusHoverState(page, '.focus-hover p-radio-button-wrapper input[type="radio"]');
-  await forceFocusState(page, '.focus-hover .force-label > p-radio-button-wrapper input[type="radio"]');
-  await forceHoverState(page, '.focus-hover .force-label > p-radio-button-wrapper >>> span');
+  await forceFocusHoverState(page, '.focus-hover p-radio-button-wrapper:not(.force-label) input[type="radio"]');
+  await forceFocusState(page, '.focus-hover p-radio-button-wrapper.force-label input[type="radio"]');
```

Comment:
Isn't this redundant with `await forceFocusState(page, '.focus p-radio-button-wrapper input[type="radio"]');`?